src/mapeo-manager.js

```diff
@@ -917,15 +919,8 @@ export class MapeoManager extends TypedEmitter {
  * @returns {PublicPeerInfo[]}
  */
 function omitPeerProtomux(peers) {
-  return peers.map(
-    ({
-      // @ts-ignore
-      // eslint-disable-next-line no-unused-vars
-      protomux,
-      ...publicPeerInfo
-    }) => {
-      return publicPeerInfo
-    }
+  return peers.map((peer) =>
+    'protomux' in peer ? omit(peer, ['protomux']) : peer
```

Comment:
The functionality is _ever so slightly_ different here. If the `protomux` property is missing, it will now return the original object instead of a copy. I don't think this causes any actual differences.